docs/source/admin/traffic_server.rst

```diff
@@ -74,54 +74,47 @@ Installing Traffic Server
 
 		#. Assuming a Traffic Monitor is already installed somewhere, check the "Cache States" table in its Web UI to verify that the :abbr:`ATS (Apache Traffic Server)` server appears.
 
-.. _traffic-ops-ort:
+.. _t3c:
 
 Configuring Traffic Server
 ==========================
-All of the :abbr:`ATS (Apache Traffic Server)` application configuration files are generated by Traffic Ops and installed by :term:`ORT`. The :file:`traffic_ops_ort.pl` file should be installed on all :term:`cache servers` (See `Installing the ORT Script`_), usually in ``/opt/ort``. It is used to do the initial install of the configuration files when the :term:`cache server` is being deployed, and to keep the configuration files up-to-date when the :term:`cache server` is already in service.
+All of the :abbr:`ATS (Apache Traffic Server)` application configuration files are generated by Traffic Ops and installed by :term:`t3c`. The :file:`t3c` file should be installed on all :term:`cache servers` (See `Installing the Cache Config Tool`_), usually in ``/opt/ort``. It is used to do the initial install of the configuration files when the :term:`cache server` is being deployed, and to keep the configuration files up-to-date when the :term:`cache server` is already in service.
 
 .. _config-generation:
 
-ORT Config File Generation
+Cache Config File Generation
 --------------------------
```

Comment:
This underline needs to be updated to be the same as the new heading's length.

Reply:
Done.